modules/using-a-pre-deployed-image-to-improve-efficiency.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/optimizing-compute-resources.adoc
+
+[id='using-a-pre-deployed-image-to-improve-efficiency-{context}']
+= Using a pre-deployed image to improve efficiency
+
+You can create a base {product-title} image with a number of tasks built-in to
+improve efficiency, maintain configuration consistency on all node hosts, and
+reduce repetitive tasks. This is known as a pre-deployed image. For example,
+because every node requires the `ose-pod` image in order to run pods, each node
+has to periodically connect to the container image registry in order to pull the
+latest image. This can become problematic when you have 100 nodes attempting
+this at the same time, and can lead to resource contention on the image
+registry, waste of network bandwidth, and increased pod launch times.
+
+.Procedure
+
+To build a pre-deployed image:
+
+. Create an instance of the type and size required.
+
+. Ensure a dedicated storage device is available for CRI-O or local image
+or container storage, separate from any persistent volumes for containers.
+
+. Fully update the system, and ensure CRI-O or Docker is installed.
+
+. Ensure the host has access to all yum repositories.
+
+. Set up thin-provisioned LVM storage.
+
+. Pre-seed your commonly used images (such as the rhel7 base image), as well as
```

Comment:
What is meant by pre-seed? I don't think this will translate well.

Reply:
pre-pull is probably a better word.  That's what is used upstream https://kubernetes.io/docs/concepts/containers/images/